packages/react-tinacms-github/src/github-client/github-client.ts

```diff
@@ -311,6 +315,30 @@ export class GithubClient {
     })
   }
 
+  async upload(
+    path: string,
+    fileContents: string,
+    commitMessage: string = 'Update from TinaCMS',
+    encoded: boolean = false
+  ) {
+    return this.githubFileApi(path, fileContents, commitMessage, encoded, 'PUT')
+  }
+
+  async delete(
+    path: string,
+    fileContents: string,
+    commitMessage: string = `Deleted ${path} using TinaCMS`,
+    encoded: boolean = false
+  ) {
+    return this.githubFileApi(
+      path,
+      fileContents,
```

Comment:
Are the file contents really needed for deleting?

Reply:
nope

Reply:
Right maybe remove `fileContents` and `encoded` from this function's API:

```ts
async delete(
  path: string
  commitMessage: string = `Deleted ${path} using TinaCMS`,
) {
```

Reply:
Not sure if this is the best of doing it. what do you think.

```ts
async delete(
    path: string,
    commitMessage: string = `Deleted ${path} using TinaCMS`,
    encoded: boolean = false
  ) {
    return this.githubFileApi(path, '', commitMessage, encoded, 'DELETE')
  }

the contents are just ignored by the GitHub API endpoint
```

Reply:
Yeah that looks good. `encoded` is also unnecessary right?